### PR TITLE
feat: Added Playlist management

### PR DESCRIPTION
**Pull request by Hardiegogo**
-  added playlist in videos context
- added create new playlist feature
- added add/delete to/from playlist functionality

Screenshots:
image

image

image

### CODE REVIEW

**Review by aman11s — APPROVED** [review]
Features are working as intended. Great work!

File: src/reducers/vidReducer.js [hunk 1]
```diff
@@ -49,6 +49,31 @@ export const vidReducer = (state, action) => {
...
+      return{
+        ...state,
+        playlists:[...action.payload]
+      }
+    case "SET_PLAYLIST":
+    case "UPDATE_PLAYLIST":
+      return{
+        ...state,
+        playlist:{...action.payload}
+      }
```
Comment (AmishMishra11) [inline src/reducers/vidReducer.js:76]: Here it is doing the same thing, so why have you created different cases, you can just use one case and one return statement

**Review by AmishMishra11 — APPROVED** [review]
Ui looks good
Features are working properly

File: src/reducers/vidReducer.js (hunk 1 above)
Comment (Hardiegogo) [reply src/reducers/vidReducer.js:76]: Got it

**Pull request merged**